indra/llwindow/llwindowwin32.cpp

```diff
@@ -3684,7 +3698,7 @@ S32 OSMessageBoxWin32(const std::string& text, const std::string& caption, U32 t
         break;
     }
 
-    int retval_win = MessageBoxW(NULL, // HWND
+    int retval_win = MessageBoxW(sWindowHandleForMessageBox, // HWND
```

Comment:
Window handle can be invalid (-1), are you certain message box will work correctly if supplies with an invalid handle?

Reply:
> Window handle can be invalid (-1)

It can't be NULL
This value is returned by CreateWindowEx
```
Return value
Type: HWND

If the function succeeds, the return value is a handle to the new window.

If the function fails, the return value is NULL. To get extended error information, call GetLastError
```
https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-createwindowexa